master-qa-14423

simonparksays opened the pull request:
https://issues.liferay.com/browse/LRQA-14423

marcellustavares committed:
LPS-54702 Remove catch clauses that make reference to DDL API, talked to Mate and if there's a hidden condition that justifies this clause it should be handled differently.

marcellustavares committed:
LPS-54702 Remove references from test classes

marcellustavares committed:
LPS-54702 Update DDMFormValuesTransformer to pass DDMFormFieldValue instance to DDMFormFieldValueTransformers

marcellustavares committed:
LPS-54702 Move verify code to be part of the upgrade process

marcellustavares committed:
LPS-54702 Update tests

izaera committed:
LPS-53136 Move MediaWiki engine to its own module

izaera committed:
LPS-53136 Move Creole engine to its own module

izaera committed:
LPS-53136 Move JSPWiki engine to its own module

zsolah committed:
LPS-55086 Refine the message shown for InvalidFolderException

Commit:
LPS-55040 Update article version properly meanwhile keeping the UUID's

Commit:
LPS-55040 Add test

Commit:
LPS-54702 Added DLFileVersion

Commit:
LPS-54702 Added ResourcePermission

Commit:
LPS-54702 Added AssetEntry

robertoDiaz committed:
LPS-54011 Create document library item selector skeleton

robertoDiaz committed:
LPS-54011 Copy and paste documents.jsp view

adolfopa committed:
LPS-55170 Access CommentManager from a unique point

robertoDiaz committed:
LPS-54011 Update classes

robertoDiaz committed:
LPS-54011 update documents.jsp

adolfopa committed:
LPS-55170 Move Comment creation logic into manager

adolfopa committed:
LPS-55170 Decouple display contexts from MBMessage

izaera committed:
LPS-53136 Create an utility JAR (non-OSGi) to host the BaseInputEditorWikiEngine class and its related resource

adolfopa committed:
LPS-55170 Decouple display contexts from MBThread

adolfopa committed:
LPS-55170 Decouple permission checks from message boards

robertoDiaz committed:
LPS-54011 Allow pass and array of Strings

adolfopa committed:
LPS-55170 Decouple display contexts from MBMessageDisplay

adolfopa committed:
LPS-55170 SF - Reorder constructor parameters

robertoDiaz committed:
LPS-54011 Copy web search classes

marcellustavares committed:
LPS-54702 SF

izaera committed:
LPS-54011 Implement correct serialization of arrays in ItemSelectorCriterions

izaera committed:
LPS-54011 SF

izaera committed:
LPS-53136 Modify wiki module to allow contribution of external wiki engines

izaera committed:
LPS-53136 Move wiki format translations from wiki-web to wiki-service

Commit:
LRQA-13970 paths

izaera committed:
LPS-53136 Adapt existing wiki engines to the changes in WikiEngine interface

I'm removing OSGi properties in favor of methods and I also define a BaseInputEditorWikiEngine class to implement those engines that share a common JSP (namely wiki.jsp).
I'm still honoring the "enabled" OSGi property to maintain compatibility. In the next commit I will remove it and use service ranking to select the proper engines for each format.

izaera committed:
LPS-53136 Fix an issue with OSGi lifecycle that caused random errors when instantiating HTMLEngine

izaera committed:
LPS-53136 Remove the "enabled" OSGi property in favor of service.ranking

As of this commit, we are not using OSGi properties any more to register wiki engines and everything is done through interface methods or standard OSGi mechanisms (namely service.ranking and the option to not deploy or stop a bundle containing a wiki engine to disable it).

adolfopa committed:
LPS-55170 Move status query methods into workflow interface

izaera committed:
LPS-53136 Move plain text engine to its own module

izaera committed:
LPS-53136 Move HTML engine to its own module

Note that the editor_config.jspf file has been copied from wiki-web

liferay-continuous-integration commented:
<h1>All tests PASSED!</h1><p>Build Time: 14 minutes 37 seconds</p><h1>See the following links for more information.</h1><ul><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest(master)/671/">https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest(master)/671/</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/671/">https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-start(master)/671/</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/667/">https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/667/</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-portalweb(master)/79/">https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-portalweb(master)/79/</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/667/">https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-githubpost-stop(master)/667/</a></li></ul>

ealonso committed:
LPS-55168 Removes portlet references on xml

ealonso committed:
LPS-55168 osgi files

ealonso committed:
LPS-55168 project files

ealonso committed:
LPS-55168 build.gradle

ealonso committed:
LPS-55168 moves jsps

ealonso committed:
LPS-55168 moves java

ealonso committed:
LPS-55168 PortletKeys

ealonso committed:
LPS-55168 moves icon

ealonso committed:
LPS-55168 Register portlet

ealonso committed:
LPS-55168 Register control panel entry

ealonso committed:
LPS-55168 upgrade

ealonso committed:
LPS-55168 Register a view portlet provider for MembershipRequest

ealonso committed:
LPS-55168 Adds servlet context

ealonso committed:
LPS-55168 init.jsp

ealonso committed:
LPS-55168 Language properties

ealonso committed:
LPS-55168 Removes old language properties

adolfopa committed:
LPS-55170 ant format-source

izaera committed:
LPS-53136 Adapt existing input-editor based engines to the new module

Commit:
LPS-55161 Having Layout Branch and Versions stored in PortalPreferences can lead to Out-of-Memory (OOM) errors

Commit:
LPS-55161 Refactor

Commit:
LPS-55161 Refactor

Commit:
LPS-55161 Change layout set branch to JSON and refactor

ealonso committed:
LPS-55168 Removes struts path for select team

ealonso committed:
LPS-55168 Implements new method in portlet provider

adolfopa committed:
LPS-55170 Don't pass around the root comment

izaera committed:
LPS-54011 Use JSONFactoryUtil to serialize criterions instead of a hand-made parameter serialization based on field names

topolik committed:
LPS-54599 Move p_auth token into POST request body for non-upload requests

Commit:
LPS-55161 SF

shuyangzhou committed:
LPS-53136 Add osgi.compile.jsp.wab.dependencies to wiki-web

Commit:
LRQA-14960 Add methods to validate else and then elements

Commit:
LPS-55170 We shouldn't expose isApproved and isPending methods in the API. getStatus method should be enough

ealonso committed:
LPS-55168 moves select team to the proper portlet

ealonso committed:
LPS-55168 Creates two new providers for Site Teams

ealonso committed:
LPS-55168 Creates a new method to get the portlet URL

ealonso committed:
LPS-55168 Uses the new method

topolik committed:
LPS-54599 define the enctype on the server-side to prevent regressions

brianchandotcom committed:
LPS-53136 SF

brianchandotcom committed:
LPS-53136 SF

brianchandotcom committed:
LPS-53136 SF

brianchandotcom committed:
LPS-53136 SF

brianchandotcom committed:
LPS-53136 rename, jsp folders go with _ vs. -

brianchandotcom committed:
LPS-53136 rename dirs, Marcellus convinced me to drop the -web except for actual apps. See modules/apps/dynamic-data-mapping and look for bnd.bnd with Web-Context*

brianchandotcom committed:
LPS-53136 regen Eclipse

brianchandotcom committed:
LPS-53136 auto SF

brianchandotcom committed:
LPS-53136 fix IDE

brianchandotcom committed:
LPS-53136 regen IDE

brianchandotcom committed:
LPS-53136 rename packages in wiki-engine-html, wiki-engine-mediawiki, and wiki-engine-text. You had already kept wiki-engine-jspwiki without the web package

brianchandotcom committed:
LPS-53136 update bnd for wiki-engine-html, wiki-engine-mediawiki, wiki-engine-text, and wiki-engine-jspwiki.

brianchandotcom committed:
LPS-53136 rename wiki-engine-creole

brianchandotcom committed:
LPS-53136 update wiki-engine-creole build.xml

brianchandotcom committed:
LPS-53136 update grammar

brianchandotcom committed:
LPS-53136 sort grammar imports

brianchandotcom committed:
LPS-53136 regen

Commit:
SYNC-1345 Deleting a folder erroneously shows a notification indicating files were added

Commit:
LPS-55161 Use constant

Commit:
LRQA-14960 Add a method to validate while element

Commit:
LRQA-15353 Update testcase to test-case

lesliewong92 committed:
LRQA-14420 Add new method to get logger elements for vars

lesliewong92 committed:
LRQA-14420 Modify to account for global variables

Ithildir committed:
LPS-51081 XSD Builder Gradle plugin - project files

lesliewong92 committed:
LRQA-14420 Modify _getLoggerElementFromElement to account for changes

Ithildir committed:
LPS-51081 XSD Builder Gradle plugin

Ithildir committed:
LPS-51081 XSD Builder Gradle plugin - sample

Ithildir committed:
LPS-51081 Use XSD Builder plugin

Ithildir committed:
LPS-51081 Rename

Ithildir committed:
LPS-51081 SF

Ithildir committed:
LPS-51081 Use Gradle utilities

kenjiheigel committed:
LRQA-15416 Update MobileDriver to use additional LiferaySeleniumHelper methods in poshi runner

brianwulbern committed:
LRQA-15398 Turn tests on

sharonchoi committed:
LRQA-13970 SF

Ithildir committed:
LPS-51081 Move configuration into task

brianchiu committed:
LRQA-15426 Add new paths for elements interacted with during a user's first login

Ithildir committed:
LPS-51081 Apply Javadoc Formatter plugin

Ithildir committed:
LPS-51081 Rename

Ithildir committed:
LPS-51081 Use Gradle utilities

Ithildir committed:
LPS-51081 Always get/set parameters on the Args bean, for consistency

Ithildir committed:
LPS-51081 "formatWSDL" and "formatXSD" tasks

Ithildir committed:
LPS-51081 Rename

Ithildir committed:
LPS-51081 Use Gradle utilities

Commit:
LPS-55227 - Permissions UI breaks: some controls are out of border

Ithildir committed:
LPS-51081 Move configuration into task

Ithildir committed:
LPS-51081 SF

Commit:
LPS-55227 - fix fieldset and input field overflow issue in firefox

lesliewong92 committed:
LRQA-14420 Overload method to get child container logger element with child content

brianchiu committed:
LRQA-15426 Add new open URL macros

brianchiu committed:
LRQA-15426 Add new sign in and sign out macros

koor4356 committed:
LRQA-15424 paths

sharonchoi commented:
Pushed paths back to SPark:
-please add //label[contains(.,'${label_of_checkbox_here}')]/ to xpaths in Checkbox.path
@simonkpark

Ithildir committed:
LPS-51081 Rename

Ithildir committed:
LPS-51081 Use Gradle utilities

Ithildir committed:
LPS-51081 Move configuration into task

brianchiu committed:
LRQA-15426 Add new go to page macros

brianchiu committed:
LRQA-15426 Have portal smoke test use new macros

brianchiu committed:
LRQA-15426 Portal smoke test should assert user is properly signed in

Commit:
LPS-55227 - Remove max-width since Bootstrap 3 is setting width to 100% already; Use display table-cell for firefox fieldset.

LeventeHudak committed:
LPS-54996 Proper doAsGroupId for the site links

izaera committed:
LPS-53136 SF

izaera committed:
LPS-53136 Make HTML wiki engine use wiki-engine-input-editor-common to get advantage of the common code in that module

Commit:
LPS-54011 SF

ealonso committed:
LPS-55168 Creates a new taglib for group search form

Commit:
LPS-54011 Make sure that we read the groupId in case that it's changed by group_selector.jsp

Commit:
LPS-54011 Simplify

Commit:
LPS-54011 Source formatting

adolfopa committed:
LPS-55170 The threadId is not strictly required

We can obtain it from the parent message, which for discussions will
never be null.

Commit:
LPS-55086 SF. Use LocalService instead of remote service

vicnate5 committed:
LRQA-15438 tests

vicnate5 committed:
LRQA-15438 macros

Commit:
LRQA-14411 Ant task to validate Poshi syntax and context

Commit:
LPS-55161 Create upgrade process

Commit:
LPS-55161 SF

brianchandotcom committed:
LPS-51081 SF

brianchandotcom committed:
LPS-51081 proper var name

brianchandotcom committed:
LPS-51081 Prep next version

brianchandotcom committed:
LPS-51081 Ivy cache

brianchandotcom committed:
LPS-51081 Use com.liferay.gradle.util 1.0.1 release

brianchandotcom committed:
LPS-51081 Forgot to increment

brianchandotcom committed:
LPS-51081 cast

brianchandotcom committed:
LPS-51081 safer

brianchandotcom committed:
LPS-51081 Increment to prepeare for next versions

brianchandotcom committed:
LPS-51081 Use latest

brianchandotcom committed:
LPS-51081 Ivy cache

brianchandotcom committed:
LRQA-15402 logic if dest is a dir and not a file

brianchandotcom committed:
LRQA-15402 sdk

brianchandotcom committed:
Record

brianchandotcom committed:
LPS-53136 Consistency

brianchandotcom committed:
LPS-53136 we should really add a bnd for this file. Instead of just copying its resources, we should find a way to delegate to this app?

jonathanmccann committed:
LPS-55161 Source Formatting

brianchandotcom committed:
LPS-55168 Regen

brianchandotcom committed:
Record

Ithildir committed:
LPS-51081 Consistency

Ithildir committed:
LPS-51081 SF

Commit:
LPS-55377 add a centralized way to register and later unregister services

Commit:
LPS-55377 add test

Commit:
LPS-53802-InvokerFilterContainerImplTest Integraion point test

brianchandotcom committed:
LPS-54011 not needed?

brianchandotcom committed:
LPS-54011 sort

brianchandotcom committed:
LPS-54011 sort

brianchandotcom committed:
LPS-54011 SF

brianchandotcom committed:
LPS-54011 Hugo, please fix? Thx.

brianchandotcom committed:
LPS-54011 SF

brianchandotcom committed:
LPS-54011 SF

brianchandotcom committed:
LPS-54011 Consistency with test

brianchandotcom committed:
LPS-54011 warning

brianchandotcom committed:
LPS-54011 SF

shuyangzhou committed:
LPS-54887 The copy on write CAS operations are too heavy. Consider the empty indexedCacheKeys set clean up does not happen very often, change the solution to post-makeup fix.

willnewbury committed:
LPS-55353 Change the level of logs when testing to allow for debugging/tracing output from catalina

michaelhashimoto committed:
LRQA-15402 Rename some the properties for mirrors get

shuyangzhou committed:
LPS-54887 Fix test

shuyangzhou committed:
LPS-55353 SF

shuyangzhou committed:
LPS-55353 Wrong key

shuyangzhou committed:
LPS-55353 Move target to correct build file

brianchandotcom committed:
LRQA-15402 sdk

Ithildir committed:
LPS-51081 SF

CAustin582 committed:
LPS-55362 Update release date for 7.0 M5

Ithildir committed:
LPS-51081 Use WSDD Builder plugin

Commit:
SYNC-1345 SF

Ithildir committed:
LPS-55043 SF

michaelhashimoto committed:
LRQA-14411 Use 'prepare-poshi-runner' in order to validate 'test.name'

brianchandotcom committed:
LPS-55043 remove unused

brianchandotcom committed:
LPS-51081 prep next release

brianchandotcom committed:
LPS-51081 Ivy cache

brianchandotcom committed:
LPS-55353 SF

brianchandotcom committed:
LPS-55358 sdk

brianchandotcom committed:
Record

brianchandotcom committed:
Ivy cache

LeventeHudak committed:
LPS-54996 Cleaning up doAsGroupId

juliocamarero committed:
LPS-55341 remove keys not used anymore

juliocamarero committed:
LPS-55341 use request instead of locale so that the language.properties from the module are also read

Commit:
LPS-55161 SF

Commit:
LPS-54599 SF

juliocamarero committed:
LPS-55341 add missing mvc paths

Commit:
LPS-55161 Parameter var is already "namespaced" by the method

Commit:
LPS-55161 SF

Commit:
LPS-55377 add way to finalize reference before deleting

Commit:
LPS-54843 Accept broader type

juliocamarero committed:
LPS-55306 fix logic

Commit:
LPS-54843 Update existing code

Commit:
LPS-54843 Update registry

Commit:
LPS-54843 Add SuppressWarnings

brianchandotcom committed:
LPS-54599 SF

brianchandotcom committed:
LPS-54996 SF

brianchandotcom committed:
LPS-55040 SF

brianchandotcom committed:
LPS-55161 SF

brianchandotcom committed:
LPS-55161 Camel case preference keys

brianchandotcom committed:
LPS-55161 SF

brianchandotcom committed:
LPS-54843 SF

Commit:
LPS-55377 fix bug in impl of getServices(Class, String)

brianchandotcom committed:
LPS-54702 Not used anymore?

brianchandotcom committed:
LPS-54702 SF

brianchandotcom committed:
LPS-54702 Can I sort this?

Commit:
LPS-55377 ensure finalize can access registration props

brianchandotcom committed:
LPS-54843 SF

Commit:
LRQA-14411 Change so validate only validates syntax and context and not if the test case name

brianchandotcom committed:
LPS-55170 SF

Commit:
LRQA-14423 paths

michaelhashimoto committed:
LRQA-14421 Combine pushStackTrace and pushFilePath into 1 method to simplify usage

michaelhashimoto committed:
LRQA-14421 Add startStackTrace to record the testcase into the stack

michaelhashimoto committed:
LRQA-14421 Add a private method '_pushFilePath' to get rid of duplicate logic

brianchandotcom committed:
LPS-55170 SF

brianchandotcom committed:
LPS-55170 SF

brianchandotcom committed:
LPS-55170 SF

brianchandotcom committed:
LPS-55170 SF

brianchandotcom committed:
LRQA-14960 SF

brianchandotcom committed:
LRQA-14420 avoid using Validator.isNotNull except for strings

brianchandotcom committed:
LPS-55377 regen

brianchandotcom committed:
LPS-55377 Generics

brianchandotcom committed:
LPS-55170 SF

brianchandotcom committed:
LPS-55377 SF

brianchandotcom committed:
LRQA-14421 SF

Commit:
LRQA-14423 macros

Commit:
LRQA-14423 tests

sharonchoi commented:
SF for SPark:
-renamed macro names
@simonkpark

sharonchoi commented:
Pushed to BChiu:
https://github.com/brianchiu/liferay-portal/pull/258
